templates/dag/dag.dag.condor.sub

```diff
@@ -31,6 +31,10 @@ delegate_job_GSI_credentials_lifetime = 0
 {%endif%}
 {{lines|join("\n")}}
 
+{% if generate_email_summary %}
+notify= {{email_to}}
```

Comment:
Actually - hang on, I see other strings are quoted in the submission file.  Does this also need to be quoted?

Reply:
Yes, it probably should be.  Stuff sometimes kind of works when you forget, but...